packages/pigeon/e2e_tests/test_objc/ios/Runner/MyApi.m

```diff
@@ -6,7 +6,10 @@
 #import <Flutter/Flutter.h>
 
 @implementation MyApi
-- (ACSearchReply *)search:(ACSearchRequest *)input error:(FlutterError **)error {
+- (void)initialize:(FlutterError *_Nullable *_Nonnull)error {
+}
+
+- (ACSearchReply *)searchRequest:(ACSearchRequest *)input error:(FlutterError **)error {
```

Comment:
Is the fact that the first argument is always called "input" legacy from single-arity? We should fix that as part of the breaking change. (Sorry, I didn't notice this before because I was focused on the signature itself.)

Reply:
I just realized that in the test code I was looking at at the end of the review, the first argument was correctly named. Is this manually written code that's following the (previous) conventions of the generator?

Reply:
Yea, the objc compiler doesn't enforce that argument names match between the header and the implementation.  In the generated header the argument name matches the dart declaration.  You're probably right that this was written before I started using the argument names in the generator.